qdrant-landing/content/documentation/concepts/search.md

```diff
@@ -776,3 +776,119 @@ If the `group_by` field of a point is an array (e.g. `"document_id": ["a", "b"]`
 
 * Only string and integer (signed and unsigned) fields are supported for the `group_by` parameter. Payload fields with other types will be ignored.
 * At the moment, pagination is not enabled when using **groups**, so the `offset` parameter is not allowed.
+
+### Lookup in groups
+
+*Available as of v1.3.0*
+
+One way of optimizing storage when using groups is to store the information shared by the points with the same group id in a single point in another collection. Then, when using the **groups** API, add the `with_lookup` parameter to bring the information from those points into each group.
```

Comment:
The doc starts right away by offering an optimization to a storage problem without actually introducing it.

I think it would be clearer  to first introduce the storage problem and then how the `lookup` feature can tackle it.

Reply:
Nice one ☝️, not assuming it is key

Reply:
Added a better intro